INFO     root:test_case_temp.py:25 开始执行用例无效登录
INFO     root:run_case.py:14 用例开始执行...
INFO     root:operation_yaml.py:34 正在调用接口关联读取方法...
INFO     root:SendRequest.py:17 请求参数详情:{'url': 'http://localhost:8888/user/login', 'method': 'POST', 'headers': {'Content-Type': 'application/json', 'User-Agent': 'python-requests/2.18.4'}, 'json': {'userName': 'admin1', 'userPassword': 'zhangbo1234'}}
INFO     root:common_assert.py:13 正在调用断言公共方法...
INFO     root:common_assert.py:24 实际结果：403
INFO     root:common_assert.py:25 期望结果：403
INFO     root:common_assert.py:24 实际结果：用户名或密码错误
INFO     root:common_assert.py:25 期望结果：用户名或密码错误
INFO     root:run_case.py:25 用例执行通过